src/app/app-loading.pug

```diff
@@ -1,4 +1,3 @@
-#spash(style='height: 100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
-  div(style='width: 80%; max-width: 150px')
+#spash(style='z-index:100; background:#fff; position:fixed; bottom: 0; right: 0; left: 0; top: 0; height:100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')  
```

Comment:
#spash or #splash?

Reply:
It's `#spash`, probably a typo from the previous commits that Michael did.

Reply:
I mean the correct id is #splash, right? We should update it

Reply:
The correct id is `#spash` (as in it is defined that way in the CSS files), while the correct name should be `splash`. It should be handled in another issue.